### ISSUE TITLE
get segmentation fault in :transform

### ISSUE THREAD

**Issue by donmikel** [tarantool/gis#6]
Test case to reproduce [here](https://gist.github.com/BHMJ/be96055a67496d5bf3d3ae2ade2c4855)

**Comment by rtsisyk** [comment]
Should be fixed by https://github.com/tarantool/gis/pull/12

**Comment by Totktonada** [comment]
Nope, it segfaults on tarantool 2.2.0-295-g544d648c9 with proj-6.0.0. I increased the iterations count to 81920 to reproduce it.
